config/routes.rb

```diff
@@ -0,0 +1,56 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [84/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)